torch/utils/collect_env.py

```diff
@@ -146,7 +151,7 @@ def get_nvidia_driver_version(run_lambda):
     return run_and_parse_first_match(run_lambda, smi, r'Driver Version: (.*?) ')
 
 
-def get_gpu_info(run_lambda):
+def get_nvidia_gpu_info(run_lambda):
```

Comment:
Please don't delete any existing functions, as this might be considered a public API change